types.go

```diff
@@ -0,0 +1,42 @@
+package gerrit
+
+import (
+	"encoding/json"
+	"errors"
+	"strconv"
+)
+
+
```

Comment:
@opalmer fixed / is fixing this in https://github.com/andygrunwald/go-gerrit/pull/38/commits/5578108000bab2db9a234108c92073807b2cda80